js-core/homeworks/homework-14/src/main.js

```diff
@@ -0,0 +1,167 @@
+parseStr(/%%APP_DIR%%/gmi, 'fixtures/src');
+parseStr(/%%APPNAME%%/gmi, 'app.js');
+parseStr(/'/gmi, '"');
+parseStr(/""/gmi, '"');
+
+console.log(someWebpackModule);
+
+/*
+ TASK - 2
+ Сделайте разметку как скриншоте используя HTML
+ вам необходимо использовать тэги(!)
+*/
+
+const task2 = true;
+
+
+/*
+TASK 3
+ JavaScript =>
+  Создать объект с методами, которые будут динамически генерировать DOM
+  Это будет тест, который мы будем разрабатывать в следующих заданиях.
+  Сейчас вам нужно только динамически создать html,
+  методы должны храниться в одном объекте.
+  Изначально на странице должен быть только <body>,
+  вызывая методы объекта нужно создать dom-элементы
+*/
+
+const quizz = {
+    create() {
+        const wraper = document.createElement('div');
+        const quizzBlock = document.createElement('div');
+        const quizTitle = document.createElement('header');
+        const quizzFooter = document.createElement('footer');
+        const quizzList = document.createElement('ol');
+        
+        wraper.setAttribute('class', 'wraper');
+        quizzBlock.setAttribute('class', 'quizz-block');
+        quizzList.setAttribute('class', 'quizz-list');
+        quizTitle.setAttribute('class', 'quizz-title');
+        quizTitle.textContent = 'Test of Programming';
+        
+        const questionTitles = ['What is the best program language?', 'What is your name?', 'What is the best car?'];
+        const answers = [
+            ['JS', 'JavaScript', 'HTML'],
+            ['Vasya', 'Petr', 'Vitya'],
+            ['Lada Sedan', 'Juguli', 'Priora']
+        ];
+        
+        function fillQuizzList() {
+            for(let i = 0; i < questionTitles.length; i++) {
+                const li = document.createElement('li');
+                const header = document.createElement('header');
+                const main = document.createElement('main');
+                
+                li.setAttribute('class', 'question-block');
+                header.setAttribute('class', 'question-title');
+                main.setAttribute('class', 'question-body');
+                header.textContent = questionTitles[i];
+                
+                for(let j = 0; j < answers[i].length; j++) {
+                    const label = document.createElement('label');
+                    const input = document.createElement('input');
+                    const p = document.createElement('p');
+                    
+                    label.setAttribute('class', 'answer');
+                    p.textContent = answers[i][j];
+                    input.setAttribute('type', 'checkbox');
+                    label.appendChild(input);
+                    label.appendChild(p);
+                    main.appendChild(label);
+                }
+                
+                li.appendChild(header);
+                li.appendChild(main);
+                quizzList.appendChild(li);
+            }
+        }
+        fillQuizzList();
```

Comment:
it looks really imperative, your code says **what you want to receive** instead of **what-to-do**

If I'm looking at such function I can't really understand the what that function going to do without looking inside.
I'll show the difference at class